client/src/components/CarsFilter/index.tsx

```diff
@@ -0,0 +1,302 @@
+import {
+  useState, SetStateAction, useEffect,
+} from 'react';
+import './style.css';
+
+function CarsFilter() {
+  const mileRang = [
+    {
+      value: 0,
+      label: '0',
+    },
+    {
+      value: 25,
+      label: '250K',
+    },
+    {
+      value: 50,
+      label: '500K',
+    },
+    {
+      value: 75,
+      label: '750K',
+    },
+    {
+      value: 100,
+      label: '1M',
+    },
+  ];
+  const cars = [
+    {
+      brand: 'Toyota',
+      model: 'Land Cruiser VXR 4.0L V6',
+      price: 48000,
+      year: 2022,
+      mileage: 36000,
+      quality: 95,
+      isGoodPrice: false,
+      location: 'palestine - gaza',
+      state: 'on-market',
+      transmission: 'auto',
+      features: [
+        'Rear entertainment screens',
+        'Parking sensors',
+        'Cruise Control',
+        'Steering wheel control',
+        'Navigation',
+        'Bluetooth',
+        'Brake assist',
+      ],
+      description: 'This car had an accedent in the right side but the quality of the body generally is good',
+      fuel: 'diesel',
+      customerId: 1,
+    },
+    {
+      brand: 'BMW',
+      model: 'X6 M - Sport',
+      price: 55000,
+      year: 2021,
+      mileage: 21000,
+      quality: 85,
+      isGoodPrice: true,
+      location: 'palestine - ramallah',
+      state: 'on-market',
+      transmission: 'manual',
+      features: [
+        'Reversing Camera',
+        'Parking sensors',
+        'Collision warning system',
+        'Navigation',
+      ],
+      description: 'This car had an accedent in the left side but the quality of the body generally is good',
+      fuel: 'petrol',
+      customerId: 2,
+    },
+    {
+      brand: 'Mercedes',
+      model: 'GLC300 2.0L I4 TC',
+      price: 58000,
+      year: 2021,
+      mileage: 15800,
+      quality: 90,
+      isGoodPrice: true,
+      location: 'palestine - alnazreth',
+      state: 'on-market',
+      transmission: 'auto',
+      features: [
+        'Rear mirror camera',
+        'Brake assist',
+        'Reversing Camera',
+        'Collision warning system',
+        'Navigation',
+      ],
+      description: 'This car had an accedent in the left side but the quality of the body generally is good',
+      fuel: 'petrol',
+      customerId: 3,
+    },
+    {
+      brand: 'Infiniti',
+      model: 'QX70 3.7L V6',
+      price: 16500,
+      year: 2015,
+      mileage: 191000,
+      quality: 70,
+      isGoodPrice: false,
+      location: 'palestine - gaza',
+      state: 'under-check',
+      transmission: 'auto',
+      features: [
+        'Navigation',
+        'Bluetooth',
+        'Brake assist',
+        'Collision warning system',
+        'Auto dimming mirror',
+        'Headsup display',
+        'Automated Parking',
+        'Adaptive cruis control',
+      ],
+      description: '',
+      fuel: 'petrol',
+      customerId: 2,
+    },
+  ];
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const brands = ['Toyota',
```

Comment:
We already have a `brand.json` file, So try to use data from there, also create a file for `models` and export data from there